### PR TITLE
ZEPPELIN-4376, ZEPPELIN-4379 Zeppelin Web Vue - Design as per Mocks and Recycle Bin

### PR DESCRIPTION
**Pull request by malayhm**
### What is this PR for?
- ZEPPELIN-4376 Design change as per new mockups (except Notebook view)
- Added Search button placeholder (actual functionality will be implemented with Foldering)
- ZEPPELIN-4379 Recycle Bin: Restore Note, Delete Permanently
- A different view for Deleted Note
- Fixed the bug which used to move the open note to the error state post note operations

Thanks, @kmaynk for the mockups.
Mockups: https://www.figma.com/file/F0m6EJLjoVmxFuYfM2FTUe/Mocks-V1

### What type of PR is it?
Feature

### What is the Jira issue?
* https://issues.apache.org/jira/browse/ZEPPELIN-4376
* https://issues.apache.org/jira/browse/ZEPPELIN-4379

### How should this be tested?
* Start the zeppelin server
* Go to the zeppelin-web-vue folder
* Run npm install (first time)
* Run npm run serve to run Zeppelin web using vue.js on localhost:8081 ( can be changed through .env file)

### Screenshots (if appropriate)
Screen Shot 2019-10-15 at 3 00 57 AM
Screen Shot 2019-10-15 at 3 01 26 AM
Screen Shot 2019-10-15 at 3 01 48 AM
Screen Shot 2019-10-15 at 3 08 40 AM

### Questions:
* Does the licenses files need update? - No
* Is there breaking changes for older versions? - No
* Does this needs documentation? - No

### CODE REVIEW

**Comment by zjffdu** [comment]
@malayhm Could you create a separated ticket for this task ? So that we can track what each PR is doing.

**Comment by malayhm** [comment]
> @malayhm Could you create a separated ticket for this task ? So that we can track what each PR is doing.

@zjffdu I have created a new JIRA for the Recycle Bin and updated the title.

**Comment by zjffdu** [comment]
Thanks @malayhm

File: zeppelin-web-vue/src/components/Notebook/Controls.vue [hunk 1]
```diff
@@ -52,52 +56,73 @@
       <a
         href="javascript: void(0);"
         @click="showDeleteConfirm"
+        v-if="!isDeleted"
       >
         <a-tooltip placement="top">
           <template slot="title">
-            <span>Delete</span>
+            <span>Move to Recycle Bin</span>
```
Comment (prabhjyotsingh) [inline zeppelin-web-vue/src/components/Notebook/Controls.vue]: `Recycle Bin` is very windows specific, should we call it trash?

Comment (malayhm) [reply zeppelin-web-vue/src/components/Notebook/Controls.vue]: Renamed Recycle Bin to Trash.

**Comment by zjffdu** [comment]
Thanks @malayhm will merge if no more comments

**Pull request closed without merging**